src/init/features/dataconnect/index.ts

```diff
@@ -113,10 +107,12 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
     cloudSqlDatabase = await promptOnce({
       message: `What ID would you like to use for your new database in ${cloudSqlInstanceId}?`,
       type: "input",
-      default: `dataconnect`,
+      default: `fdc-db`,
```

Comment:
Changed to `fdcdb` now. See https://github.com/firebase/firebase-tools/pull/7243